src/main/java/com/gmail/inverseconduit/commands/sets/CoreBotCommands.java

```diff
@@ -31,6 +31,8 @@
 
     private static final Pattern     javadocPattern = Pattern.compile("^" + Pattern.quote(BOT_CONFIG.getTrigger()) + "javadoc (.*)", Pattern.DOTALL);
 
+    private static final Pattern     TELL_PATTERN   = Pattern.compile("^" + Pattern.quote(BOT_CONFIG.getTrigger()) + "tell (.*?) (.*)$");
```

Comment:
Eh, this accepts `"~tell  hi"`, with two continuous spaces instead of a proper user. Why not  `"tell ([^ ]+) (.*)"` instead?